pilot/cmd/istioctl/main.go

```diff
@@ -484,11 +482,8 @@ istioctl context-create --api-server http://127.0.0.1:8080
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&platform, "platform", "p", kubePlatform,
 		"Istio host platform")
-	defaultKubeconfig := path.Join(homedir.HomeDir(), ".kube/config")
-	if v := os.Getenv("KUBECONFIG"); v != "" {
-		defaultKubeconfig = v
-	}
-	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "c", defaultKubeconfig,
+
+	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "c", "$KUBECONFIG else $HOME/.kube/config",
```

Comment:
This is a functional regression (see https://github.com/istio/istio/pull/2365). For the purposes of doc generation can you explicitly set KUBECONFIG to a non-empty value, i.e. `KUBECONFIG=.kube/config istioctl collaterall --all`?